portfolio/src/main/webapp/script.js

```diff
@@ -30,7 +30,43 @@ function findZoe() {
   const location = locations[Math.floor(Math.random() * locations.length)];
 
   // Add it to the page.
-  const link = document.getElementById("link");
+  const link = document.getElementById('link');
   link.innerText = location;
   link.href = locationToLink[location];
 }
+
+function grow() {
+  const srcToAlt = {
+    'images/basil1.jpg': 'Basil Day 0',
+    'images/basil2.jpg': 'Basil Day 10',
+    'images/basil3.jpg': 'Basil Day 20',
+    'images/basil4.jpg': 'Basil Day 30',
+    'images/basil5.jpg': 'Basil Day 40',
+    'images/basil6.jpg': 'Basil Day 60',
+  };
+  const image = document.getElementById('basil');
+  var imgIndex = 1;
+  const imgScr = `images/basil${imgIndex}.jpg`;
+  console.log(`Let's grow some basil`);
+
+  // Recursive call
```

Comment:
Curious on why you made this recursive. Why is it cause you want it to try it out using recursion or was it because the combination of a for loop and the timeout function didn't do what you expect it to do?